CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 Changelog
 =============
 
-## 2016/05/11 - Version 0.1
+## 2016/05/12 - Version 0.1
```

Comment:
If we are changing to version  0.2-snapshot, shouldn't we have the same version here?